converters/fromZigbee.js

```diff
@@ -698,6 +698,44 @@ const converters = {
             }
         },
     },
+    ZNCLDJ11LM_curtain_genAnalogOutput_change: {
+        cid: 'genAnalogOutput',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            let running = false;
+            if (msg.data.data['61440']) {
+                if ((msg.data.data['61440'].toString()).startsWith('117440')
+            ||(msg.data.data['61440'].toString()).startsWith('591')) {
+                    running = true;
+                } else if (msg.data.data['61440']===0) {
+                    running = false;
+                }
+
+                return {presentValue: precisionRound(msg.data.data['presentValue'], 2), running: running};
+            }
+            return {presentValue: precisionRound(msg.data.data['presentValue'], 2),
+                running: running};
+        },
+    },
+    ZNCLDJ11LM_curtain_closuresWindowCovering_change: {
+        cid: 'closuresWindowCovering',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            if (msg.data.data['currentPositionLiftPercentage']) {
+                const lift = precisionRound(msg.data.data['currentPositionLiftPercentage'], 2);
+                return {currentPositionLiftPercentage: lift};
+            }
+        },
+    },
+    ZNCLDJ11LM_genBasic_change: {
+        cid: 'genBasic',
+        type: 'attReport',
+        convert: (model, msg, publish, options) => {
+            if (msg.data.data['65281']) {
+                return {summaryReport: msg.data.data['65281']};
```

Comment:
Whats inside this?

Reply:
The contents of this is: {"3":30,"5":13,"7":"0x0000000000000000","8":4361,"9":256,"100":0}
I could not figure out what those values mean, also they do not change while moving the curtain.
I therefore set the "genBasic attReport" to ignored now.